staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -170,7 +170,7 @@ func pushHandler(ctx context.Context, twin *common.Twin, client *driver.Customiz
 	}
 	reportCycle := time.Duration(twin.Property.ReportCycle)
 	if reportCycle == 0 {
-		reportCycle = common.DefaultReportCycle
+		reportCycle = 1 * time.Second
```

Comment:
We change it to common.DefaultReportCycle in #5230  
```suggestion
		reportCycle = common.DefaultReportCycle
```

Reply:
yes, I pushed the new code.